system/globalid/lnuid.cpp

```diff
@@ -87,7 +89,7 @@ namespace ln_uid {
 #if defined(__linux__) || defined(__APPLE__)
         FILE *fp;
         fp = fopen("/dev/urandom", "r");
-        fread(&randomdata, 1, random_byte_count, fp);
+        IGNORE_RETURN_VALUE(fread(&randomdata, 1, random_byte_count, fp));
```

Comment:
Feels like it would be better to just check that the return value actually matches what is expected...